core/platforms/packaging/bareos.spec

```diff
@@ -356,19 +356,28 @@ Group:      Productivity/Archiving/Backup
 Requires:   %{name}-common = %{version}
 Provides:   %{name}-fd
 %if 0%{?suse_version}
-Requires(pre): pwdutils
+Requires(pre): shadow
 %else
 Requires(pre): shadow-utils
 %endif
 
 %package    common
 Summary:    Common files, required by multiple Bareos packages
 Group:      Productivity/Archiving/Backup
-Requires:   openssl
+Requires(pre): coreutils
+Requires(pre): findutils
+Requires(pre): gawk
+Requires(pre): grep
+Requires(pre): openssl
+Requires(pre): sed
 %if 0%{?suse_version}
-Requires(pre): pwdutils
+Requires(pre): glibc
+Requires(pre): shadow
+#Requires(pre): libcap-progs
 %else
+Requires(pre): glibc-common
 Requires(pre): shadow-utils
+#Requires(pre): libcap
```

Comment:
we can probably remove that instead of keeping it commented out
Or - if you really want to keep it - add a comment why it is commented out :)